src/windows/access-helpers.c

```diff
@@ -0,0 +1,56 @@
+#include "access-helpers.h"
+
+static BOOL frida_access_is_windows_vista_or_greater (void);
+static BOOL frida_access_is_windows_8_or_greater (void);
+static BOOL frida_access_is_windows_version_or_greater (DWORD major, DWORD minor, DWORD service_pack);
+
+LPCWSTR
```

Comment:
Nitpick: I suppose this one could be `const`? (To hint at it not being a freshly allocated string.)